ci-operator/config/openshift-kni/eapol-operator/openshift-kni-eapol-operator-main.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  authenticator-base:
+    name: "4.14"
+    namespace: ocp
+    tag: base-rhel9
+  builder:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.14
+  operator-base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  project_image:
+    dockerfile_path: images/build-root/Dockerfile
+images:
+- dockerfile_path: Dockerfile
+  from: operator-base
+  inputs:
+    builder:
+      as:
+      - golang:1.20
```

Comment:
consider either using a stream
https://docs.ci.openshift.org/docs/architecture/images/#controlling-go-versions-in-component-repositories

or going all the way for a repo managed ci-operator.yaml
https://docs.ci.openshift.org/docs/architecture/images/#controlling-go-versions-in-component-repositories

Reply:
or maybe even better to use project_image
https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image

Reply:
Since I'm already using a custom project_image for the build_root, I don't actually need a separate 'builder' - Latest changes reuses that same 'root' image for the builder of both main images, and the runtime env for the 'make test' later on.